src/lesson7/taskNumber3/QA27_7_3_Lugovskiy.java

```diff
@@ -0,0 +1,36 @@
+package lesson7.taskNumber3;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class QA27_7_3_Lugovskiy {
+    public static void main(String[] args) {
+        System.out.print("Введите колличество животных : ");
+        Scanner scanner = new Scanner(System.in);
+        Animals[] animal = new Animals[scanner.nextInt()];
+        for (int i = 0; i < animal.length; i++) {
+            animal[i] = getNextAnimal();
+        }
+
+        for (int i = 0; i < animal.length; i++) {
+            animal[i].fly();
```

Comment:
Sense was to filter entities, who can do that, not to throw error for some of them